src/Client/Web/Bit.Client.Web.BlazorUI/Components/Pivot/BitPivot.razor.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class BitPivot
+    {
+        protected override string RootElementClass => "bit-pvt";
+
+        private string selectedKey;
+        private OverflowBehavior _overflowBehavior = OverflowBehavior.None;
```

Comment:
as in our non-written conventions :), it is not required to prepend these property fields with an underscore.